data.tf

```diff
@@ -5,7 +5,7 @@ data "aws_ami" "tableau" {
     name = "name"
 
     values = [
-      "dq-ops-win-tab-dev-*",
+      "dq-ops-win-tab-dev- 2023-01-10T16*",
```

Comment:
There's a space before the year.
Is that correct?

Reply:
Yep, unfortunately https://github.com/UKHomeOffice/dq-packer-ops-win-tableau-dev/blob/feature/yel6151/packer.json#L38 (old file), this has been updated on the packer.json to get rid of that space now https://github.com/UKHomeOffice/dq-packer-ops-win-tableau-dev/blob/master/packer.json#L40